scss/util/_color.scss

```diff
@@ -114,7 +114,7 @@ $foundation-colors: (
 /// @returns {Color} color from foundation-colors
 @function get-color($key) {
   @if map-has-key($foundation-colors, $key) {
-    @return map-get($foundation-colors, $key);
+    @return map-safe-get($foundation-colors, $key);
```

Comment:
The point of using `map-safe-get` is to not having to check if the key exist or throwing an error. `get-color($key)` is an alias for `map-safe-get($foundation-colors, $key)`.